packages/sdk/src/index.js

```diff
@@ -0,0 +1,22 @@
+import * as BudibaseApi from "../sdk"
```

Comment:
This is great cheeks, so good to have the api docs generating our client like this. 

I was wondering would there be a more fluent way we could use this client, from the docs above we have:
```js
import { configure, ApplicationsApi } from "@budibase/sdk"
```
I think it would be really nice if we could import it like 
```js
import Budibase from "@budibase/sdk"
```


Similarly for the usage, atm we have this:
```js
await ApplicationsApi.applicationsSearchPost
```

It could be more fluent to use a single import for budibase and not then need to pull in the individual api clients. So it could look something like this:
```js
await Budibase.applications.search()
await Budibase.applications.delete()
await Budibase.queries.execute()
```

I'm a fan of this approach as it requires less manual intervention with imports when I want to use different parts of the sdk and seems to be more consistent with api clients i've used from npm. It would reduce some duplication in calling code 

e.g. **Applications**Api.**applications**SearchPost could be condensed by a single `applications` namespace on the client. 


What are your thoughts on this type of usage?

Reply:
Yea it would be super easy to do a default export of everything, which solves the first point. You could do it already with `import * as Budibase from "@budibase/sdk"` but a default import is probably nicer. I'll add that. 

For the naming of the different APIs themselves, that's also totally in our control. Happy to go for a terser naming convention here so I'll update that.

The one I'm a little more on the fence about is how explicit we are with the actual endpoint names. Personally I don't mind being explicit with these, but I agree that shorter looks nicer of course. Just an FYI that we do at least have them renamed to `searchRows` for example rather than `rowsSearchPost` which is the default! But I'm happy to go either way on this and let other people weigh in too.

Reply:
👍 for the single, top level import. Since we have a single call to `configure`, it's easier to reason about IMO if we have one top level object that everything hangs off. This also very closely maps to a lot of our internal stuff as well. That said I'm also not against:

`Budibase.searchApplications()`
`Budibase.deleteApplication()`
`Budibase.executeQuery()`

Reply:
That `delete` is so annoying, we've ran into it a few times. The `call` bit might be troublesome for people relying on intelisense, as if they were to type the characters `de` looking for `delete` they're less likely to come across the right function.

Airtable uses a `destroy` function, and stripe uses `del`. I assume they've been hit by the same problem.

`destroy` might be the best alternative here as `del` might look strange in anything other than a js client e.g. generated docs.  I'm also aware that I've asked you for many changes at this point and would like to apologise in advance

Reply:
`destroy` sounds good to me :ok_hand: and don't worry - this is exactly the kind of feedback/suggestions I was after!

Reply:
> That `delete` is so annoying, we've ran into it a few times. The `call` bit might be troublesome for people relying on intelisense, as if they were to type the characters `de` looking for `delete` they're less likely to come across the right function.
> 
> Airtable uses a `destroy` function, and stripe uses `del`. I assume they've been hit by the same problem.
> 
> `destroy` might be the best alternative here as `del` might look strange in anything other than a js client e.g. generated docs. I'm also aware that I've asked you for many changes at this point and would like to apologise in advance

Updated to use `destroy` for delete operations :+1: